### PR TITLE
 Added El Jalisco. Again.

### PR DESCRIPTION
**Pull request by cquinlan**

### CODE REVIEW

**Comment by cquinlan** [comment]
It is I who was the coward all along.

**Comment by Dman757** [comment]
coward.

**Comment by Dman757** [comment]
@qpfiffer merge this you coward.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 *.swp
 *.pyc
+.vscode/settings.json
```
Comment (qpfiffer) [inline .gitignore:3]: WTF r u doing

**Comment by qpfiffer** [comment]
Sure whatever kid :package:

File: .gitignore (hunk 1 above)
Comment (Dman757) [reply .gitignore:3]: don't invalidate our life choices

**Pull request merged**